tests/main/ubuntu-core-create-user/task.yaml

```diff
@@ -7,6 +7,15 @@ restore: |
     sed -i '/^mvo/d' /var/lib/extrausers/group
     rm -rf /home/mvo
 execute: |
+    if [ "$MANAGED_DEVICE" = "true" ]; then
```

Comment:
@niemeyer thanks for your review! I added these changes because the create user test began to fail using the external backend on ubuntu-core devices with an error message stating that the user couldn't be added because the device was already managed. So for these devices we could just skip the test or check for the failure.

AIUI, please correct me if I'm wrong, we cannot have the same test for both kind of systems (managed and unmanaged), because we currently are unable to issue `snap create-user` on managed systems. The `MANAGED_DEVICE` flag allows us to know which kind of test to run (positive or negative), now assuming that all the systems in the external backend are managed. Maybe we could get rid of it if we can detect if the device is managed using another method, please let me know what do you think.